litmus-portal/frontend/src/components/InputField/styles.ts

```diff
@@ -0,0 +1,25 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  success: {
+    border: '0.0625rem solid',
+    borderColor: theme.palette.secondary.dark,
+  },
+
+  error: {
+    border: '0.0625rem solid',
+    borderColor: theme.palette.error.main,
+  },
+
+  inputArea: {
+    width: '20rem',
```

Comment:
You can use theme spacing instead of 20rem directly

Reply:
Actually, it was mentioned in the guidelines to use rem for height and widths and for margin we can use theme spacing.

Reply:
theme.spacing goes in multiples of 8 so might not be the best candidate for width and heights....for padding and marine it's ok

Reply:
I guess we can go with this as it has been mentioned in the guidelines